config/config.go

```diff
@@ -771,9 +771,9 @@ func DefaultP2PConfig() *P2PConfig {
 		UPNP:                          false,
 		AddrBook:                      defaultAddrBookPath,
 		AddrBookStrict:                true,
-		MaxNumInboundPeers:            40,
-		MaxNumOutboundPeers:           10,
-		MaxConnections:                64,
+		MaxNumInboundPeers:            120,
+		MaxNumOutboundPeers:           30,
+		MaxConnections:                150,
```

Comment:
I do not have any confidence in these settings tbh.

Reply:
Me either. We can revert these.

Reply:
reverted, but fwiw I originally increased because of https://github.com/tharsis/evmos/pull/541/files and https://github.com/osmosis-labs/osmosis/pull/464